Resources/doc/annotations.rst

```diff
@@ -252,3 +252,85 @@ Automatically, registers the given class as a listener with the Doctrine ORM or
 
     ``@DoctrineListener`` implies ``@Service`` if not explicitly defined.    
 
+
+Custom Annotations
+~~~~~~~~~~~~~~~~~~
+Build your own annotations and register them
+
+.. code-block :: php
+
+    <?php
+
+    namespace AppBundle\Annotation;
+
+    use JMS\DiExtraBundle\Annotation\DoctrineListener;
+    use JMS\DiExtraBundle\Metadata\ClassMetadata;
+
+    /**
+     * @Annotation
+     */
+    class Special implements MetadataProcessorInterface
+    {
+        public function processMetadata(ClassMetadata $metadata)
+        {}
+    }
+
+register your annotations, either as prepend extension or by adding it to your config.yml
```

Comment:
I'm not sure that we should specify that people can prepend the bundle configuration in their extensions.
This is not what people should do on a project except if they are developing generic bundles.